packaging/linux/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM debian
+FROM debian:10
```

Comment:
`debian:stretch` is what we use everywhere else

Reply:
sure thing, I thought we'd go with latest because that's what it defaulted to